abstract.tex

```diff
@@ -113,15 +114,15 @@ \section*{Abstract:}
 small portion of the cargo space.
 
 To that end, we have developed a compact reaction wheel that fits in the cargo
-space of a standard cargo bicycle capable of applying roll torques to the
-vehicle of up to 200~\si{\newton\meter}, see Figure~\ref{fig:cad-model}. This
-can can stabilize the roll degree of freedom at zero forward speed for lean
-angles up to about 10~\si{\degree}.
+space of a standard cargo bicycle. The reaction wheel is capable of applying
+roll torques up to 200~\si{\newton\meter} to the vehicle, see
+Figure~\ref{fig:cad-model}. This can stabilize the roll degree of freedom at
+zero speed for roll angles up to about 10~\si{\degree}.
 
 Figure~\ref{fig:initial-value-simulation} shows that the reaction wheel can
 stabilize the vehicle from a 2~\si{\degree} disturbance within a second using
 about 200~\si{\joule} and 100~\si{\newton\meter} of peak torque. A modern
```

Comment:
@moorepants Given these numbers, a rough calculation gives (4 MJ) / (200 (J / s)) =
5.55555556 hours. Do these numbers make sense or is there something else we can mention such that the need to minimize energy consumption is more apparent?